### PR TITLE
Feature/service infrastructure improvements

### PR DESCRIPTION
**Pull request by santee**

### CODE REVIEW

File: server/Arcadia.Assistant.Web/Startup.cs [hunk 1]
```diff
@@ -104,6 +107,8 @@ public void ConfigureServices(IServiceCollection services)
                         policy.RequireAuthenticatedUser();
                     });
                 });
+
```
Comment (sponkrashin) [inline server/Arcadia.Assistant.Web/Startup.cs]: Not needed! Very critical!

File: server/Arcadia.Assistant.Web/Infrastructure/WebTelemetryInitializer.cs [hunk 2]
```diff
@@ -0,0 +1,13 @@
+﻿namespace Arcadia.Assistant.Web.Infrastructure
+{
+    using Microsoft.ApplicationInsights.Channel;
+    using Microsoft.ApplicationInsights.Extensibility;
+
+    public class WebTelemetryInitializer : ITelemetryInitializer
+    {
+        public void Initialize(ITelemetry telemetry)
+        {
+            telemetry.Context.Cloud.RoleName = "Arcadia Assistant Web";
```
Comment (sponkrashin) [inline server/Arcadia.Assistant.Web/Infrastructure/WebTelemetryInitializer.cs]: Doesn't make sense to have it in settings?

Comment (santee) [reply server/Arcadia.Assistant.Web/Infrastructure/WebTelemetryInitializer.cs]: no

File: server/Arcadia.Assistant.Web/Startup.cs (hunk 1 above)
Comment (santee) [reply server/Arcadia.Assistant.Web/Startup.cs]: FIXED!

**Pull request merged**